patient_portal/messaging/secure_chat.php

```diff
@@ -975,10 +975,10 @@ public function pingAction()
                                         ng-class="{'pull-left':!message.me, 'pull-right':message.me}">{{message.date }}</span>
                                 </div>
                                 <img class="direct-chat-img" ng-show="!message.me"
-                                    src="<?php echo $GLOBALS['images_path']; ?>/Unknown-person.gif"
+                                    src="<?php echo $GLOBALS['images_path']; ?>/unknown-person.gif"
```

Comment:
Case sensitive check needed here.  Make sure the file name resolves on Linux server (it still would on Windows).